packages/ingest/src/db-helpers/step.ts

```diff
@@ -0,0 +1,30 @@
+import { IngestError } from "../IngestError";
+
+export const STEP = [
```

Comment:
This is the order that an ingest happens in.
What stops it from being completely linear is that there are two batch processes for which we have to poll for status from OpenAI.